api/nodes/views.py

```diff
@@ -1390,8 +1390,6 @@ def get_object(self):
             self.kwargs[node_link_lookup_url_kwarg],
             'node link'
         )
-        # May raise a permission denied
-        self.check_object_permissions(self.request, node_link)
```

Comment:
This doesn't seem like the proper way to do this. I think modifying what permission classes are on the view would be a better idea than removing the object permissions check altogether. Same with registrations below. We do a lot of things in permission checking, and this is a little to broad of a change.